packages/pie/src/Pie.js

```diff
@@ -28,191 +23,183 @@ import PieSlicesLabels from './PieSlicesLabels'
-            enableSlicesLabels,
-            getSliceLabel,
-            slicesLabelsSkipAngle,
-            slicesLabelsTextColor,
-
-            // styling
-            theme,
-            defs,
-            fill,
-
-            // interactivity
-            isInteractive,
-            onClick,
-            onMouseEnter,
-            onMouseLeave,
-            tooltipFormat,
-            tooltip,
-
-            legends,
-            role,
-        } = this.props
-
-        const borderColor = getInheritedColorGenerator(_borderColor, theme)
-
-        return (
-            <PieLayout
-                width={width}
-                height={height}
-                data={data}
-                sortByValue={sortByValue}
-                startAngle={startAngle}
-                endAngle={endAngle}
-                fit={fit}
-                padAngle={padAngle}
-                innerRadius={innerRadius}
-                cornerRadius={cornerRadius}
-                colors={colors}
-                colorBy={colorBy}
-            >
-                {({ centerX, centerY, radius, innerRadius, arcs, arcGenerator }) => {
-                    const boundDefs = bindDefs(defs, arcs, fill, {
-                        dataKey: 'data',
-                    })
-
-                    return (
-                        <Container isInteractive={isInteractive} theme={theme} animate={false}>
-                            {({ showTooltip, hideTooltip }) => (
-                                <SvgWrapper
-                                    width={outerWidth}
-                                    height={outerHeight}
-                                    margin={margin}
-                                    defs={boundDefs}
+export default function Pie(props) {
+    const {
+        data,
+        sortByValue,
+
+        startAngle,
+        endAngle,
+        padAngle,
+        fit,
+        innerRadius,
+        cornerRadius,
+
+        // dimensions
+        margin: _margin,
+        width: _width,
+        height: _height,
+
+        colors,
+        colorBy,
+
+        // border
+        borderWidth,
+        borderColor: _borderColor,
+
+        // radial labels
+        radialLabel,
+        enableRadialLabels,
+        radialLabelsSkipAngle,
+        radialLabelsLinkOffset,
+        radialLabelsLinkDiagonalLength,
+        radialLabelsLinkHorizontalLength,
+        radialLabelsLinkStrokeWidth,
+        radialLabelsTextXOffset,
+        radialLabelsTextColor,
+        radialLabelsLinkColor,
+
+        // slices labels
+        sliceLabel,
+        enableSlicesLabels,
+        slicesLabelsSkipAngle,
+        slicesLabelsTextColor,
+
+        // styling
+        // theme,
+        defs,
+        fill,
+
+        // interactivity
+        isInteractive,
+        onClick,
+        onMouseEnter,
+        onMouseLeave,
+        tooltipFormat,
+        tooltip,
+
+        legends,
+        role,
+    } = props
+
+    const theme = usePartialTheme()
```

Comment:
We can pass a `theme` prop to the component, so we need to override the default theme with those values. You can see this in the `custom tooltip` story. The background of the tooltip should be black and it is white.

Reply:
I fixed this.  I also noticed that in `props.js` that theme isn't a required or optional prop listed in the `propTypes` object.  I don't know if this was done intentionally, but I wanted to bring it up.

Reply:
It looks like it is present on PieLegends, might as well make it available for the main components as well.

Reply:
Hmm, where do the prop types for the pie specific theme reside?  I looked in `nivo/packages/core/theming/propTypes.js` (since that is where PieLegends gets the propTypes declaration from), but it doesn't have the `tooltip` property that I see is being used inside of the stories.  It looks like `theme.tooltip` is only ever defined via prop types in `PieTooltip.js`.  I could of course do something like:
```js
import { themePropType } from '@nivo/core'
export const PiePropTypes = {
    // other prop types 
    theme: PropTypes.shape({ ...themePropType, tooltip: PropTypes.shape({}) })
}
```
However, I'm not sure if that encompasses all of the custom theming for the pie package.


Maybe this is starting to get outside of the scope of this PR.